Tests/Model/MessageTest.php

```diff
@@ -115,7 +117,7 @@ public function testMergeRememberDesc()
         $message->merge($message2);
 
         $this->assertEquals('foo_desc', $message->getDesc());
-        $this->assertEquals('foo_desc', $message->getLocaleString());
+        $this->assertEquals('foo_id', $message->getLocaleString());
```

Comment:
Why was this test changed? 

IMHO the line does not bring any value at all anyways. I removed this line.